test/mocks/users.js

```diff
@@ -0,0 +1,68 @@
+const { UNPROCESSABLE_ENTITY, BAD_REQUEST } = require('../../app/errors');
+const {
+  CREATED,
+  VALIDATE_PASSWORD_MATCHES,
+  VALIDATE_EXISTS,
+  validationMessage
+} = require('../../config/constants');
+
+const passwordInvalid = '111av';
+
+const createUserInput = {
+  name: 'Julián',
+  lastName: 'Barrientos',
+  email: 'julian1.lopera@wolox.co',
+  password: 'password1'
+};
+
+const createUserOutput = {
+  data: {
+    name: createUserInput.name
+  },
+  message: CREATED
+};
+
+const uniqueEmailErrorMessage = 'email must be unique';
```

Comment:
you have this message in the constants file, haven't you?

Reply:
This string is not found in `constanst.js`, because `${field} must be unique` is returned from _sequelize_ when `User.create` function is called.

Reply:
ok, cool ! 😎